src/game/combat.ts

```diff
@@ -26,5 +26,12 @@ const dealCombatDamage = (
   from: InstanceState | null,
   extraDamage: number,
 ): void => {
-  dealDamage($, to, getAttribute($, from, "ATTACK") + extraDamage, from);
+  const runeModifier = from?.plusMinusRunes ?? 0;
```

Comment:
Shouldn't plusMinusRunes just be included in `getAttribute($, from, "ATTACK")`, if it's not already?

Reply:
Dunno what I was thinking at the time, but looking at it now I think that makes a lot of sense.